content/en/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -35,28 +35,22 @@ components still rely on Endpoints. For now, enabling EndpointSlices should be
 seen as an addition to Endpoints in a cluster, not a replacement for them.
 {{< /note >}}
 
-EndpointSlices are considered a beta feature, but only the API is enabled by
-default. Both the EndpointSlice controller and the usage of EndpointSlices by
-kube-proxy are not enabled by default.
+EndpointSlices are considered a beta feature. Both the API and EndpointSlice
+Controller are enabled by default. The usage of EndpointSlices by kube-proxy is
+not enabled by default.
```

Comment:
```suggestion
EndpointSlices are a beta feature. Both the API and the EndpointSlice
{< glossary_tooltip term_id="controller" >}} are enabled by default.
{{<  glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}
uses Endpoints by default, not EndpointSlices.
```

Reply:
It's “controller” not “Controller” because there isn't a Kubernetes object called Controller that you can manage via `kubectl`.

Reply:
Updated, thanks!